acceptance_tests/features/environment.py

```diff
@@ -76,6 +75,9 @@ def after_scenario(context, scenario):
     if 'UI' in context.tags:
         context.browser.quit()
 
+    if purge_outbound_topics():
+        test_helper.fail('There are left over messages on some topics that were not expected.')
```

Comment:
I think we need the test to tell us the subscription of any messages, and ideally log out the message contents too.

Reply:
agreed this would be much more useful for debugging and trying to find patterns